test/regressions/index.tsx

```diff
@@ -80,7 +80,7 @@ const tests: Test[] = [];
 
 // Also use some of the demos to avoid code duplication.
 // @ts-ignore
-const requireDocs = require.context('docsx/data', true, /\.js$/);
+const requireDocs = import.meta.glob(['docsx/data/**/[A-Z]*.js'], { eager: true });
```

Comment:
`import: 'default'` does not work here since we have some data files (i.e. dataset for charts) which have only named exports. 🤷